Attributions/Attributions/AttributionSection.swift

```diff
@@ -4,10 +4,12 @@ public struct AttributionSection {
 
     public let file: URL
     public let description: String
+    public let additionalAttributions: [Attribution]?
 
-    public init(file: URL, description: String) {
+    public init(file: URL, description: String, additionalAttributions: [Attribution]? = nil) {
```

Comment:
We could maybe make it non-optional with a default of `[]` then you won't have to unwrap anything on use. I always feel like for collections the empty collection is the same as the `nil` collection just that you can normally use the empty collection without having to worry about optionality.